### ISSUE TITLE
Incorrect instruction

### ISSUE THREAD

**Issue by karenpayneoregon** [dotnet/docs#36284]
The instructions do not add the class to the project and keeps the original namespace.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: d9cb6bd4-1194-91fa-014c-2e72d809e772
* Version Independent ID: c3fddda9-24c1-f20c-a1b6-6aeea61d6e53
* Content: [Create an item template for dotnet new - .NET CLI](https://learn.microsoft.com/en-us/dotnet/core/tutorials/cli-templates-create-item-template)
* Content Source: [docs/core/tutorials/cli-templates-create-item-template.md](https://github.com/dotnet/docs/blob/main/docs/core/tutorials/cli-templates-create-item-template.md)
* Product: **dotnet-fundamentals**
* GitHub Login: @adegeo
* Microsoft Alias: **adegeo**

---
[Associated WorkItem - 147954](https://dev.azure.com/msft-skilling/Content/_workitems/edit/147954)

**Comment by adegeo** [comment]
Hello. I'll take a look at this in Sept. Thanks for the info.

**Comment by adegeo** [comment]
Hi @karenpayneoregon I'm having a problem figuring out what you're referencing. Are you talking about the section where you install the item template then try to use it?

**Comment by karenpayneoregon** [comment]
I would tell you but do not remember the page link.

**Comment by karenpayneoregon** [comment]
Thanks, will respond back later today, head is in a different place right now.

**Comment by karenpayneoregon** [comment]
Well since it has been too long since I submitted this I can't remember what was incorrect.

What I do remember was getting better information from
[https://github.com/sayedihashimi/template-sample](https://github.com/sayedihashimi/template-sample)

There (and I don't remember) part of template.json that needed to be included that were not. The following was taken from my [GitHub repository](https://github.com/karenpayneoregon/Templating-dotnet), ignore the postActions they work but do remember that was not an issue.

```json
{
    "$schema": "http://json.schemastore.org/template",
    "author": "Karen Payne",
    "classifications": [
        "Web",
        "Razor"
    ],
    "identity": "KarenRazor",
    "name": "Razor from Karen",
    "shortName": "karenrazorproj",
    "tags": {
        "language": "C#",
        "type": "project"
    },
    "sourceName": "KarenRazor",
    "preferNameDirectory": "true",
    "primaryOutputs": [
        {
            "path": "KarenRazor.csproj"
        },
        {
            "path": "readme.md"
        }
    ],
    "postActions": [
        {
            "description": "Opens README.md in the editor",
            "manualInstructions": [],
            "actionId": "84C0DA21-51C8-4541-9940-6CA19AF04EE6",
            "args": {
                "files": "1"
            },
            "continueOnError": true
        }
    ]
}
```

So in closing being its been so long I don't remember what was wrong on the page just that using those instructions did not work will the code in my [GitHub repository](https://github.com/karenpayneoregon/Templating-dotnet) does work.

**Comment by adegeo** [comment]
OK. Thanks for checking. I was just a bit confused because the item template (article this issue was filed on) doesn't change the namespace because the example is an extension method which needs to keep the "System" namespace. Additionally, classes don't need to be added to projects because any code file in the directory (or below) of a project is automatically included in the project.

It's possible that you were looking at the next step in the tutorial which was creating a project template, but even that doesn't specify a namespace (using the default for whatever your project is named).

Regardless! I used this issue as a way to go into the template articles and update them. I ran through the first 2 (item and project templates) articles and everything worked as-is. I'm going to update the code slightly to use some of the newer C# formatting and also just little textual issues I found I'm fixing.

Really the tutorial series needs to be extended with more articles that describe these other options. I'm also adding a link to the reference wiki page: https://github.com/dotnet/templating/wiki/Reference-for-template.json
